src/i18n/es.js

```diff
@@ -174,6 +174,13 @@ export default {
     verified_domain: `Última visita registrada:`,
     week_with_plural: `{weeksCount, plural, =0 {sin semanas} one {# semana}other {# semanas} }`,
   },
+  reports_hours_visits: {
+    few_visits: `0 a 300`,
+    lot_visits: `600 a 1k`,
```

Comment:
Here perhaps from 600 onwards right? what happens if you have more than 1k visits a day?

Reply:
maybe need change the content

Reply:
ready